app/controllers/api/account.php

```diff
@@ -1677,6 +1677,96 @@
         ])]), Response::MODEL_JWT);
     });
 
+App::post('/v1/account/targets')
+    ->desc('Create Account Target')
+    ->groups(['api', 'account'])
+    ->label('error', __DIR__ . '/../../views/general/error.phtml')
+    ->label('audits.event', 'target.create')
+    ->label('audits.resource', 'target/response.$id')
+    ->label('event', 'users.[userId].targets.[targetId].create')
+    ->label('scope', 'public')
```

Comment:
Let's remove this and add session auth, should only be accessible if you have a current session

Reply:
added session auth